src/blue.ts

```diff
@@ -234,13 +235,16 @@ export function blueProxyFactory(env: MembraneBroker) {
             if (isUndefined(redDescriptor)) {
                 // looking in the blue proto chain to avoid switching sides
                 const blueProto = getBlueValue(ReflectGetPrototypeOf(target));
+                if (isNull(blueProto)) {
+                    return;
```

Comment:
Consistency note: In some places we are explicit and return `undefined` others the implicit `return;` is used.

Reply:
+1